### ISSUE TITLE
Add support for FILESAVER engine

### ISSUE THREAD

**Issue by Theverat** [LuxCoreRender/BlendLuxCore#33]
* [x] Text format
* [ ] Binary format

**Comment by Theverat** [comment]
implemented in a2ed27ec17ed6bf4d364a353ef8a7fd6c7e5308f

**Comment by Dade916** [comment]
I have added the support for saving in binary format to FILESAVER. The new parameters are described here: https://wiki.luxcorerender.org/LuxCore_SDL_Reference_Manual_v2.0#.22FILESAVER.22

You can stil save in text format with:

```
renderengine.type = FILESAVER
# This is optional because "TXT" is the default
filesaver.format = TXT
filesaver.directory = <directory name>
filesaver.renderengine.type = PATHCPU
```

as before and in binary format:

```
renderengine.type = FILESAVER
filesaver.format = BIN
filesaver.filename = <file name>
filesaver.renderengine.type = PATHCPU
```

note the usage of "filesaver.filename" for BIN format and "filesaver.directory" for TXT format. In BlendLuxCore, you can just use "<directory name>/render.bcf" for "filesaver.filename" if you don't want to have an different input field.
